### PR TITLE
Avoid ReferenceError window is not defined in the node environment

### PR DESCRIPTION
**Pull request by MartinNuc**
To solve [issue with server side rendering](https://github.com/flowjs/ngx-flow/issues/7) I need flow.js to be able to survive missing `window` and `document` objects.

I am not sure how to cover this by tests however :-( any suggestions?

### CODE REVIEW

File: src/flow.js [hunk 1]
```diff
@@ -2,6 +2,9 @@
  * @license MIT
  */
 (function(window, document, undefined) {'use strict';
+  if (!window || !document) {
+    console.warn('Flowjs needs window and document objects to work');
+  }
   // ie10+
   var ie10plus = window.navigator.msPointerEnabled;
```
Comment (AidasK) [inline src/flow.js]: This is still going to break with an error here. Shouldn't we call `return;` after console.warn?

Comment (MartinNuc) [reply src/flow.js]: Oh, god how could I? :-) Of course you are right. When I tested it I had return. Then I put there the warning but forgot to keep the return. Going to update PR.

**Comment by AidasK** [comment]
Merged. Do you need a build in dist/ too?

**Comment by AidasK** [comment]
Nm, made a new build released under 2.13.1

**Comment by MartinNuc** [comment]
Ya I need it also in dist. Thank you.

**Comment by MartinNuc** [comment]
@AidasK I just tried that and it looks like the 2.13.1 is not published to npm repository: https://www.npmjs.com/package/@flowjs/flow.js

Can you please publish it? I could probably publish it on my own but don't want to mess anything :-)

**Comment by AidasK** [comment]
Published. I haven't done this either, but hope it works

**Pull request merged**